pkg/services/featuremgmt/registry.go

```diff
@@ -413,6 +413,11 @@ var (
 			State:       FeatureStateAlpha,
 			Owner:       grafanaBackendPlatformSquad,
 		},
+		{
+			Name:        "onlyExternalOrgRoleSync",
```

Comment:
naming things! anyone feel like they have a good suggestion?

We want to make the name as a opt-in